src/store/reducers/rootReducer.js

```diff
@@ -13,6 +15,7 @@ import { firebaseReducer } from "react-redux-firebase";
 
 const rootReducer = combineReducers({
   auth: authReducer,
+  contact: contactReducer,
   firestore: firestoreReducer,
   firebase: firebaseReducer, // Authentication
```

Comment:
Comment should not be in the same line.

Reply:
This is okay, once it explains the content of an Object directly.

```
var x = 5;       // Declare x, give it the value of 5
var y = x + 2;   // Declare y, give it the value of x + 2
```